src/Page/Partners/Partner_.elm

```diff
@@ -193,6 +194,15 @@ viewAddress maybeAddress =
         Nothing ->
             p [ css [ contactItemStyle ] ] [ text (t PartnerAddressEmptyText) ]
 
+partnerLogo : Maybe String -> Html msg
+partnerLogo maybeLogoUrl =
+  case maybeLogoUrl of
+    Just logoUrl ->
+      img [ src logoUrl ] []
+    Nothing ->
+      div [] [text "no logo set"]
```

Comment:
This maybe wants a `[cCc] or todo so it doesn't accidentally make it into production.

Any thought on whether we want to change layout or use a placeholder image here @kimadactyl . We should check what design specifies?

Reply:
yeah no image if there's none specified